webpack.config.js

```diff
@@ -58,6 +59,23 @@ const modeConfig = production
 
 const config = merge(modeConfig, withTarget)
 
+const removeCSSMQPackerPlugin = config => {
```

Comment:
I know that you know and you'll remove those lines after an cozy-scripts upgrade, but hey adding a link to the issue and adding a todo comment can be great for external ppl.
```suggestion
//TODO remove those lines after https://github.com/cozy/create-cozy-app/pull/1326 is fixed
const removeCSSMQPackerPlugin = config => {
```